stories/Callout/Callout.stories.js

```diff
@@ -0,0 +1,217 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+
+import { withA11y } from '@storybook/addon-a11y'
+import { withInfo } from '@storybook/addon-info'
+
+import { Callout, CalloutText, CalloutTitle, Icon } from '../../src'
+
+import Esempio from './docs/Esempio.md'
+import EsempioConIcona from './docs/EsempioConIcona.md'
+import EsempioScreenReader from './docs/EsempioScreenReader.md'
+import ClasseSuccess from './docs/ClasseSuccess.md'
+import ClasseWarning from './docs/ClasseWarning.md'
+import ClasseDanger from './docs/ClasseDanger.md'
+import ClasseImportant from './docs/ClasseImportant.md'
+import ClasseNote from './docs/ClasseNote.md'
+import ClasseHighlights from './docs/ClasseHighlights.md'
+
+const BasicExample = () => (
+  <Callout>
+    <CalloutTitle>Titolo Callout</CalloutTitle>
+    <CalloutText>
+      Maecenas vulputate ante dictum vestibulum volutpat. Lorem ipsum dolor sit
+      amet, consectetur adipiscing elit. Aenean non augue non purus vestibulum
+      varius.
+    </CalloutText>
+  </Callout>
+)
+
+const ExampleWithIcon = () => (
+  <Callout>
+    <CalloutTitle>
+      <Icon
+        className={'aria-hidden: true'}
```

Comment:
this doesn't seem ok. Do you mean `style={{ariaHidden: true}}` maybe?

Reply:
Yes, I did. Thanks for pointing it out.